.azure-pipelines/gpu-tests.yml

```diff
@@ -52,9 +52,10 @@ jobs:
 
     - bash: |
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
-        pip install fairscale==0.4.5
+        pip install fairscale>=0.4.5
         pip install deepspeed>=0.6.0
-        pip install bagua-cuda102==0.9.0
+        CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
```

Comment:
Isn't this simpler?

```suggestion
        CUDA_VERSION_MM=$(python -c "import torch ; print(torch.version.cuda.replace('.', ''))")
```

Reply:
how about if version `11.3.1`?

Reply:
By the way, what does `MM` in `CUDA_VERSION_MM` stand for? :eyes: